test/test_binary_ufuncs.py

```diff
@@ -1183,6 +1183,34 @@ def test_binary_ops_with_scalars(self, device):
                         self.assertEqual(expected, python_op(first, second))
                         self.assertEqual(expected, torch_op(first, second))
 
+    # Tests that the function and its (array-accepting) reference produce the same
+    #   values on a range of tensors, including empty tensors, scalar tensors,
+    #   1D tensors and a large 2D tensor
+    @onlyOnCPUAndCUDA
+    @ops(binary_ufuncs)
+    def test_reference_numerics(self, device, dtype, op):
+        if dtype == torch.bfloat16:
+            np_dtype = np.float16
+
+            def to_numpy(t):
+                return np.array(t.tolist(), dtype=np_dtype)
+
+        else:
+            np_dtype = torch_to_numpy_dtype_dict[dtype]
+
+            def to_numpy(t):
+                return np.array(t.cpu(), dtype=np_dtype)
+
+        # TODO: turn these into separate tests
+        for sample in op.sample_inputs(device, dtype):
+            torch_inputs = (sample.input,) + sample.args
+            numpy_inputs = [to_numpy(t) for t in torch_inputs]
+
+            torch_output = getattr(torch, op.name)(*torch_inputs).cpu()
```

Comment:
if you want to call the function variant of an op then `op` is a callable and will call the function variant:

```
torch_output = op(*torch_inputs).cpu()
```